src/user.rs

```diff
@@ -29,18 +29,49 @@ impl Default for DeviceCreateOpts {
     }
 }
 
+/// Options that can be specified at when calling [`user_create`](trait.UserOps.html#tymethod.user_create).
+pub struct UserCreateOpts {
+    // see docs on `new`
+    needs_rotation: bool,
+}
+
+impl UserCreateOpts {
+    /// Constructor. Also see [`default`](#method.default).
+    ///
+    /// # Arguments
+    /// - `needs_rotation` - Set to true if the private key for this user should be rotated when the user "takes control" of their keys.
+    ///
+    /// The main use case for this is a workflow that requires that users and groups to be generated
+    /// prior to the user logging in for the first time. In this situation, a user's cryptographic identity
+    /// can be generated by a third party, like a server process, and then the user can take control of their
+    /// keys by rotating the private key using [`rotate_private_key`](TODO LINK).
+    pub fn new(needs_rotation: bool) -> UserCreateOpts {
+        UserCreateOpts { needs_rotation }
+    }
+}
+
+impl Default for UserCreateOpts {
+    fn default() -> Self {
+        UserCreateOpts::new(false)
+    }
+}
+
 pub trait UserOps {
     /// Sync a new user within the IronCore system.
     ///
     /// # Arguments
     /// - `jwt` - Valid IronCore or Auth0 JWT
     /// - `password` - Password used to encrypt and escrow the user's private master key
-    ///
+    /// - `user_create_opts` - see [`UserCreateOpts`](struct.UserCreateOpts.html)
     /// # Returns
     /// Newly generated `UserCreateKeyPair` or Err. For most use cases this key pair can
     /// be discarded as IronCore escrows your user's keys. The escrowed keys are unlocked
     /// by the provided password.
-    fn user_create(jwt: &str, password: &str) -> Result<UserCreateKeyPair>;
+    fn user_create(
+        jwt: &str,
+        password: &str,
+        user_create_opts: &UserCreateOpts,
```

Comment:
I assume it's a purposeful choice to break this API and not make this `Option<UserCreateOpts>` instead? 

Related, should you change the version to `0.11.0` in the Cargo.toml as part of this PR?

Reply:
Yes, intentional, but good point on the version bump.